integration/nemo/examples/supervised_fine_tuning/jobs/templates/client/config/config_fed_client.json

```diff
@@ -0,0 +1,36 @@
+{
+  "format_version": 2,
+  "model_file": "/workspace/Models/nemo-megatron-gpt-1.3B/nemo_gpt1.3B_fp16.nemo",
```

Comment:
Let's try not to load the model on the client side, as it's being overwritten by the server anyhow. Leave as optional argument to allow loading the initial checkpoint if given and allow partial training of subset of the layers.

Reply:
Doesn't seem trivial, the NemoSFT model / training script is set in such a way that for initialization, it needs to restore from a checkpoint, let's keep it this way to avoid too much deviation from NeMo's script.

Reply:
I see. Might be an enhancement in the future.